docs/setup/postgresql.md

```diff
@@ -35,7 +35,7 @@ psql -d georchestra -f postgresql/080-geofence.sql
 
 ## OGC statistics schema
 
-If the **security proxy** is deployed and ```shared.ogc.statistics.activated``` is true in your setup (false by default):
+If the **security proxy** is deployed and ```OGC_STATISTICS``` is true in your setup ([true by default](https://github.com/georchestra/datadir/blob/18.06/analytics/js/GEOR_custom.js#L4)):
```

Comment:
rather: If SP's log4j uses the org.georchestra.ogcservstatistics.log4j.OGCServicesAppender as configured in https://github.com/georchestra/datadir/blob/docker-master/security-proxy/log4j/log4j.properties

This is more logical than refering to a client side property here (which is just a consequence).

Reply:
Fixed, thanks